members/taksh/data.json

```diff
@@ -0,0 +1,9 @@
+{
+  "id": "takshch",
+  "first_name": "Taksh",
+  "last_name": "Chanana",
+  "yoe": "College 2nd year student",
```

Comment:
yoe can be 0
company name can be your college name
designation can be **student**

Reply:
I don't want to be a student here!.